### PR TITLE
chore: update strong-globalize version

### PR DESCRIPTION
**Pull request by dhmlau**
### Description

Related to https://github.com/strongloop-internal/scrum-apex/issues/372
- update `strong-globalize` version
- update `CODEOWNERS` to reflect the current status.

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -1,6 +1,9 @@
...
 # Each line is a file pattern followed by one or more owners,
-# the last matching pattern has the most precendence.
+# the last matching pattern has the most precedence.
+
+# Alumni maintainers
+# @ssh24 @virkt25 @loay @kjdelisle
 
 # Core team members from IBM
-* @kjdelisle @jannyHou @loay @b-admike @ssh24 @virkt25 @dhmlau
+* @jannyHou@b-admike @dhmlau
```
Comment (bajtos) [inline CODEOWNERS]: Missing space?

```suggestion
* @jannyHou @b-admike @dhmlau
```

**Comment by dhmlau** [comment]
I think there was a change in juggler causing the downstream CI failing.  @b-admike is looking into it.

**Comment by dhmlau** [comment]
@slnode test please

**Comment by b-admike** [comment]
@slnode test please

**Comment by dhmlau** [comment]
@slnode test please

**Pull request merged**